packages/demo/examples/TSDigisolPlatform/configs/routes/routesConfig.js

```diff
@@ -299,7 +299,7 @@ routes.orderDetail = {
     },
   },
 };
-
+//demo checking
```

Comment:
Hi @rajatkhade 

Connect with @nihal772 
for this file I have reviews his file on this